rendermodules/dataimport/create_mipmaps.py

```diff
@@ -1,10 +1,12 @@
-from PIL import Image
-import argparse
 import os
+from PIL import Image
 from rendermodules.module.render_module import RenderModuleException
 
+
 class CreateMipMapException(RenderModuleException):
     """Exception raised when there is a problem creating a mipmap"""
+    pass
```

Comment:
I'm not concerned about this, just curious... is this pass actually necessary?

Reply:
I guess either that or a docstring does the trick (and docstring actually adds something), so there's not a need for both.  More force of habit than anything else.